src/backendbot/repositories/ram_repository.py

```
@@ -0,0 +1,101 @@
+            p = psutil.Process(pid)
+            p.resume()
+            return True
+        except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess):
+            return False
+
+    def kill_process(self, pid: int) -> bool:
+        """Terminates a process by PID."""
+        try:
+            p = psutil.Process(pid)
+            p.terminate()
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)